Artesian/_Query/AuctionQuery.py

```diff
@@ -0,0 +1,55 @@
+from Artesian._Query.Query import _Query
+from Artesian._Query.QueryParameters.AuctionQueryParamaters import AuctionQueryParameters
+from Artesian._Query.Config.ExtractionRangeConfig import ExtractionRangeConfig
+from Artesian._Configuration.DefaultPartitionStrategy import DefaultPartitionStrategy
+import urllib
+class _AuctionQuery(_Query):
+    __routePrefix = "auction"
+    def __init__(self, client, requestExecutor, partitionStrategy):
+        queryParameters = AuctionQueryParameters(None,ExtractionRangeConfig(), None, None, None) 
+        _Query.__init__(self, client, requestExecutor, queryParameters)
+        self.__partition= partitionStrategy
+
+    def forMarketData(self, ids):
+        super()._forMarketData(ids)
+        return self
+    def forFilterId(self, filterId):
+        super()._forFilterId(filterId)
+        return self
+    def inTimeZone(self, tz):
+        super()._inTimezone(tz)
+        return self
+    def inAbsoluteDateRange(self, start, end):
+        super()._inAbsoluteDateRange(start, end)
+        return self
+    def inRelativePeriodRange(self, pStart, pEnd):
+        super()._inRelativePeriodRange(pStart, pEnd)
+        return self
+    def inRelativePeriod(self, extractionPeriod):
+        super()._inRelativePeriod(extractionPeriod)
+        return self
+    def execute(self):
+        urls = self.__buildRequest()
+        return super()._exec(urls)
+    async def executeAsync(self):
+        urls = self.__buildRequest()
+        return super()._execAsync(urls)
+    def __buildRequest(self):
+        self.__validateQuery()
+        qps = self.__partition.PartitionAuction([self._queryParameters])
+        urls = []
+        for qp in qps:
+            url = f"/{self.__routePrefix}/{super()._buildExtractionRangeRoute(qp)}?_=1"
```

Comment:
'?_=1' Not sure of the reason for this in the url

Reply:
I think is a sort of "optimization" so that you don't need to distinguish between starting with '?' or '&' to concatenate query params.